openslides/agenda/static/templates/agenda/item-list.html

```diff
@@ -8,11 +8,14 @@
         <translate>New</translate>
       </a>
       <!-- import -->
-      <a ui-sref="topics.topic.import" os-perms="agenda.can_manage"
+      <span os-perms="agenda.can_manage">
```

Comment:
@normanjaeckel @emanuelschuetze You can only import topics correctly if you have both permissions. Without 'can_manage' you cannot set duration, weight,.. and without 'can_see_hidden_items' you cannot set the type.

Same changes in AgendaUpdate factory.

Is it useful that a person with 'can_manage' does not have 'can_see_hidden_items'?